ads/google/a4a/test/test-traffic-experiments.js

```diff
@@ -76,26 +71,26 @@ describe('all-traffic-experiments-tests', () => {
 
     it('handles empty experiments list', () => {
       // Opt out of experiment.
-      sandbox.win.AMP_CONFIG['testExperimentId'] = 0.0;
+      toggleExperiment(sandbox.win, 'testExperimentId', false, true);
```

Comment:
This changes the semantics of the test, doesn't it?  The previous semantic was, "When AMP_CONFIG says that this experiment has 0 probability, then randomly selection should not turn it on."  Here, you're forcing it off via the toggle and then testing to see that it's still off.